### ISSUE TITLE
Cannot install on 14.04 (Trusty)

### ISSUE THREAD

**Issue by mi302** [sarim/ibus-avro#57]
I didn't find a Ubuntu 14.04 version of Avro. Tried to install 12.04 and 13.10, but don't work.

**Comment by ssi-anik** [comment]
@JHShovon Yes, I've followed those steps. But, Avro Phonetic is not shown on text entry input source.
![screenshot from 2016-03-11 02 08 52](https://cloud.githubusercontent.com/assets/2676602/13682948/5644665c-e72e-11e5-92f2-3f07efb09d9e.png)
